### ISSUE TITLE
0.5.16 breaks node 8 compatibility

### ISSUE THREAD

**Issue by SoyYoRafa** [ardatan/graphql-toolkit#259]
Hi,
The latest release is using `for await` which is not available in node 8. node 8 is in still in LTS until January 2020. Would it be possible to revert back that syntax usage?

This could be as simple as bumping down the `tsconfig.json` target to `es2017`.

**Comment by janhartmann** [comment]
Yes, currently hitting this issue as well:

```toolkit/dist/commonjs/loaders/load-typedefs.js:109
        for await (let path of globby.stream(foundGlobs, { cwd, absolute: true })) {
            ^^^^^

SyntaxError: Unexpected reserved word
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/node_modules/graphql-toolkit/dist/commonjs/index.js:7:22)```

**Comment by ecwyne** [comment]
Same ☝️

**Comment by ecwyne** [comment]
For those using Zeit Now, upgrading to Node 10 was an easy fix for now. https://zeit.co/blog/node-10

**Comment by jsphweid** [comment]
Yes, would appreciate a fix. CI for our api can't go through until a platform team upgrades node versions now... and that doesn't happen instantly.

**Comment by ardatan** [comment]
Could you try with `0.5.17-ddafc4c.0`?

**Comment by mdlavin** [comment]
Using yarn's package.json feature:

```
  "resolutions": {
    "graphql-toolkit": "0.5.17-ddafc4c.0"
  }
```

fixes the problem for me.

**Comment by ardatan** [comment]
@mdlavin Sounds good!

**Comment by dotansimha** [comment]
Fixed in v0.5.17
